sound/soc/sof/imx/imx8.c

```diff
@@ -458,6 +460,9 @@ struct snd_sof_dsp_ops sof_imx8x_ops = {
 	.ipc_msg_data	= imx8_ipc_msg_data,
 	.ipc_pcm_params	= imx8_ipc_pcm_params,
 
+	/* Firmware ops */
+	.arch_ops = &sof_xtensa_arch_ops,
+
```

Comment:
don't you need this for ALL versions of i.mx? it's e.g. not added line 422 for imx_ops.

Reply:
@plbossart yes, this is true.